swagger/v1/swagger.yaml

```diff
@@ -19,7 +19,7 @@ components:
             type: string
           email:
             type: string
-          api_token_expires_at:
+          token_expires_at:
```

Comment:
Thanks 👍👍

Reply:
no problem. Just in case you weren't aware, we shouldn't edit this yaml file. It is automatically generated by the `bin/rails rswag:swaggerize` command.

So in the future whenever we add swagger request specs or make edits to the `swagger_helper.rb`
then make sure to run the above swaggerize command to update the yaml. 😎